ebr_trackerbot/storage/odbc.py

```diff
@@ -0,0 +1,44 @@
+"""
+Slack Bot ODBC Storage
+"""
+
+import pyodbc
+import logging
+from bot import register_storage, config
+import db
+from functools import partial
+
+
+def create_connection(config):
+    if config.get("odbc_connection_string") is None:
+        raise RuntimeError("Missing odbc_connection_string in configuration")
+    link = pyodbc.connect(config.get("odbc_connection_string"))
+    db.create_table(lambda: link)
+    return link
+
+
+def get_connection(config):
```

Comment:
I think this is unnecessary, as it looks like connection pooling is built-in.
https://github.com/mkleehammer/pyodbc/wiki/Module#pooling